### PR TITLE
feat: Reorder Overview Page Widgets - MEED-2860 - Meeds-io/MIPs#104

### LINKED ISSUE #104: Overview review
## Rationale

After a year of using the overview page, UX needs to be slightly improved:
- Placeholder needs to be reviewed to have consistent display between items
- Labels as well
- End-users'd rather know what actions to do than their achievements, kudos, wallet, etc.

So we suggest a slight review of this page to answer these feedbacks

### Top User Stories

Overview items are reversed so we have in first row, the information of what to do:
image

When accessing the platform and nothing has been setup yet
Then placeholders are reviewed to be more concise and with better balance between them
image

For weekly achievements portlet, then display the pie-chart as is with no subtitles as currently
image

AND when clicking to see all, then display the achievement list of the user in the drawer
image

When no kudos yet, the same placeholder is displayed in the same place no matter we have badges or not
image

And the contrary as well (kudos but no badges)
image

And an access to dedicated drawer is proposed for each (kudos creation or badges)
image

When no wallet yet, then no need to display the perkstore placeholder
image

When wallet init, but no perks, then depending on ability to create, two placeholders are proposed
- When ability to create:
image

- When no ability to create:
image

**PRECISION**

Condidering that the overview lists users, we need to make sure that username is not displayed when not logged in. Thus, the URL for avatar of users will be modified to hide the username with an anonymous ID

#### Gamification
Overview gadgets review

#### Notifications
NA 

#### Analytics
NA 

#### Unified Search
NA 

### Upgradability

The Overview page will changed to reorder the portlets. This change will be applied automatically since the import mode is `merge` of `meeds` and `global` sites.

### Feature Flags

No feature flags

### Access 

The UIs has to be enhanced without adding new portlets. (UI modifications only for better UX)

### PR DESCRIPTION
**Pull request by boubaker**
This change will reorder Overview widget by reversing rows order.

### CODE REVIEW

File: webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml [hunk 1]
```diff
@@ -645,85 +645,85 @@
...
               </portlet>
-              <title>My Contributions</title>
+              <title>Top Challengers</title>
               <access-permissions>*:/platform/users;*:/platform/externals</access-permissions>
               <show-info-bar>false</show-info-bar>
             </portlet-application>
           </container>
           <container
-            id="ReputationContainer"
+            id="ChanllengesOverviewContainer"
```
Comment (rdenarie) [inline webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml]: typo here

File: webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml [hunk 2]
```diff
@@ -26,8 +26,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       <properties-param>
         <name>MeedsPortalConfigProperties</name>
         <property name="exo.social.groups.portalConfig.metadata.importmode" value="${exo.social.groups.portalConfig.metadata.importmode:OVERWRITE}" />
-        <property name="exo.social.portalConfig.profilePage.importmode" value="${exo.social.portalConfig.profilePage.importmode:MERGE}" />
         <property name="exo.social.portalConfig.metadata.importmode" value="${exo.social.portalConfig.metadata.importmode:OVERWRITE}" />
+        <property name="io.meeds.portalConfig.metadata.importmode" value="${io.meeds.portalConfig.metadata.importmode:OVERWRITE}" />
+        <property name="io.meeds.groups.portalConfig.metadata.importmode" value="${io.meeds.groups.portalConfig.metadata.importmode:OVERWRITE}" />
```
Comment (rdenarie) [inline webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml:31]: can you list which group navigation will be overwritten ?

File: webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml [hunk 3]
```diff
@@ -26,8 +26,9 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
       <properties-param>
         <name>MeedsPortalConfigProperties</name>
         <property name="exo.social.groups.portalConfig.metadata.importmode" value="${exo.social.groups.portalConfig.metadata.importmode:OVERWRITE}" />
-        <property name="exo.social.portalConfig.profilePage.importmode" value="${exo.social.portalConfig.profilePage.importmode:MERGE}" />
         <property name="exo.social.portalConfig.metadata.importmode" value="${exo.social.portalConfig.metadata.importmode:OVERWRITE}" />
+        <property name="io.meeds.portalConfig.metadata.importmode" value="${io.meeds.portalConfig.metadata.importmode:OVERWRITE}" />
```
Comment (rdenarie) [inline webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml:30]: can you list which portal navigation will be overwritten ?

**Review by rdenarie — CHANGES_REQUESTED** [review]

File: webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml (hunk 2 above)
Comment (boubaker) [reply webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml:31]: This flag will be used for **Meeds Package** context only. In fact, all groups navigations will be overwritten each startup of Meeds Server.

File: webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml (hunk 3 above)
Comment (boubaker) [reply webapps/plf-meeds-extension/src/main/webapp/META-INF/exo-conf/configuration.xml:30]: This flag will be used for **Meeds Package** context only. In fact, all `meeds` site navigations will be overwritten each startup of Meeds Server.

File: webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml (hunk 1 above)
Comment (boubaker) [reply webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml]: Yes, thanks, it was defined in develop, [see it here](https://github.com/Meeds-io/meeds/blob/develop/webapps/plf-sites-extension/src/main/webapp/WEB-INF/conf/sites/portal/global/pages.xml#L704)

Knowing that there is no specific CSS to this id, I can manage to change it without affecting the UI. Fixed.

```suggestion
            id="ChallengesOverviewContainer"
```

**Review by rdenarie — DISMISSED** [review]

**Pull request merged**